common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -11,7 +11,16 @@
  */
 
 (function (requirejs, require, define) {
-
+require.config({
```

Comment:
As I mentioned in the ticket, I would like this requirejs config moved out into a central location so that it can easily be extended for new dependencies.

I'd also like to know the scope of this fix-- does it address TNL-3311 as well? It is OK if it does not as the 404 is more serious, but I'm not clear about the impact on TNL-3311.

Finally, take care within indenting to consistently use 4 spaces per indent.


TNL-3311